modules/viewing-stats-collected-kubernetes-nmtate-op.adoc

```diff
@@ -0,0 +1,90 @@
+// This is included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-stats-collected-kubernetes-nmtate-op_{context}"]
+= Viewing metrics collected by the Kubernetes NMState Operator
+
+The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and them as ready-to-use metrics.
+
+[NOTE]
+====
+The metrics `kubernetes_nmstate_features_applied` are not an API and might change between {product-title} versions.
+====
+
+In the *Developer* perspective, the Metrics UI includes some predefined CPU, memory, bandwidth, and network packet queries for the selected project. You can run custom Prometheus Query Language (PromQL) queries for CPU, memory, bandwidth, network packet and application metrics for the project.
+
+You can also obtain the HTP service endpoint by specifying the namespace for the `kubernetes_nmstate_features_applied` service. For example:
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a developer or as a user with view permissions for the project that you are viewing metrics for.
+* You have enabled monitoring for user-defined projects.
+* You have deployed a service in a user-defined project.
+
+.Procedure
+
+. From the *Developer* perspective in the {product-title} web console, select *Observe* -> *Metrics*.
+
+. Select the project that you want to view metrics for in the *Project:* list.
+
+. Select a query from the *Select query* list, or create a custom PromQL query based on the selected query by selecting *Show PromQL*. The metrics from the queries are visualized on the plot.
++
+[NOTE]
+====
+In the *Developer* perspective, you can only run one query at a time.
+====
+
+. Explore the visualized metrics by doing any of the following:
++
+|===
+|Option |Description
+
+|Zoom into the plot and change the time range.
+a|Either:
+
+* Visually select the time range by clicking and dragging on the plot horizontally.
+* Use the menu in the left upper corner to select the time range.
+
+|Reset the time range. |Select *Reset zoom*.
+
+|Display outputs for all queries at a specific point in time. |Hold the mouse cursor on the plot at that point. The query outputs appear in a pop-up box.
+|===
+
+.Example
+
+The following `NodeNetworkConfigurationPolicy` manifest was applied to an {product-title} cluster:
++
+[source,text]
+----
+...
+interfaces:
+  - name: br1
+    type: linux-bridge
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: true
+      dhcp-custom-hostname: foo
+    bridge:
+      options:
+        stp:
+          enabled: false
+      port: []
+...
+----
++
+The `nmstatectl st` command outputs the following example statistics, where some parts are redacted for clarity, according to the configured `NodeNetworkConfigurationPolicy` manifest:
```

Comment:
Instead of writing here that we show an output of `nmstatectl st`, we should rather write that after applying the NNCP object above, the following metrics will be exposed and available via OpenShift Monitoring. This way you keep below what you have now, but you abstract out the details because people do not need to know that `nmstatectl st` is behind, they only need to know what once they applied NNCP to the cluster, the metrics will automagically appear